### ISSUE TITLE
Lipsync_expert.pth 

### ISSUE THREAD

**Issue by Zeyad-Mohamed** [Rudrabha/Wav2Lip#252]
python inference.py --checkpoint_path <ckpt> --face <video.mp4> --audio <an-audio-source>

this line works well with wav2lip.pth and wav2lip_gan.pth which are the pretrained model in the checkpoint path but the pretrained model of the discrimnator which is "lipsync_expert.pth" results in error 

![image](https://user-images.githubusercontent.com/58393760/115302963-3da23c80-a120-11eb-9993-8104c66a6bea.png)

**Comment by 15458wew** [comment]
The inference stage  does not need  lip-sync_expert.pth

**Comment by Zeyad-Mohamed** [comment]
how come ??

**Comment by ambitionforcomputervision** [comment]
The `inference.py` script is intended for generating talking head frames. As you can see in figure 2 of the [paper](https://arxiv.org/pdf/2008.10010.pdf) it is the _Generator_ that generates talking head frames. There are two pre-trained versions of this Generator `wav2lip.pth` and `wav2lip_gan.pth`. In contrast, the _Lip Sync Expert_ generates a sync loss which is used during training to tell the generator if it is doing well. If you are interested in this loss value for a specific video you should not use `inference.py` which is intended for generating talking head frames and not a specific loss value.

**Comment by Zeyad-Mohamed** [comment]
> The `inference.py` script is intended for generating talking head frames. As you can see in figure 2 of the [paper](https://arxiv.org/pdf/2008.10010.pdf) it is the _Generator_ that generates talking head frames. There are two pre-trained versions of this Generator `wav2lip.pth` and `wav2lip_gan.pth`. In contrast, the _Lip Sync Expert_ generates a sync loss which is used during training to tell the generator if it is doing well. If you are interested in this loss value for a specific video you should not use `inference.py` which is intended for generating talking head frames and not a specific loss value.

thanks so much for your time ,, its much more clear now thank you

**Comment by Jayyy-H** [comment]
> python inference.py --checkpoint_path --face <video.mp4> --audio
> 
> this line works well with wav2lip.pth and wav2lip_gan.pth which are the pretrained model in the checkpoint path but the pretrained model of the discrimnator which is "lipsync_expert.pth" results in error
> 
> ![image](https://user-images.githubusercontent.com/58393760/115302963-3da23c80-a120-11eb-9993-8104c66a6bea.png)

hi,can you share the lipsync_expert.pth,cause the original link is no longer valid,thank you!
